src/Plugins/DBFTPlugin/Consensus/ConsensusService.Check.cs

```diff
@@ -46,8 +46,8 @@ private bool CheckPrepareResponse()
                 }
 
                 // Timeout extension due to prepare response sent
-                // around 2*15/M=30.0/5 ~ 40% block time (for M=5)
-                ExtendTimerByFactor(2);
```

Comment:
dotnet-format?

Reply:
I was unable to do that because of devcontainer problems

Reply:
use `mcr.microsoft.com/dotnet/sdk:9.0.101-noble`

Reply:
@cschuchardt88 I just created the PR for that...now it is possible

Reply:
In fact, this image you mentioned does not exist yet in the repo. I tried with devcontainer but the only one working for me was that one merged in the PR.

Reply:
It does exist and works. You need the `SDK`

![image](https://github.com/user-attachments/assets/16a261a6-140d-4b25-bd7c-d9a0406ca6d3)

Reply:
sure, now it works better